src/api/app/models/changes_file.rb

```diff
@@ -0,0 +1,14 @@
+# Backend::File model to represent files that belongs to the package in the backend
+class ChangesFile < PackageFile
+  # You dont want to change name of .changes
+  private :name=
+
+  def initialize(params = {})
+    super
+    @name = '.changes'
+  end
+
+  def full_path(query = {})
+    URI.encode("/source/#{project_name}/#{package_name}/#{package_name}#{name}")
```

Comment:
Discussed offline. No need to handle `multibuild` here, better in the controller...